docs/developer_guide/Continous_Integration.md

```diff
@@ -0,0 +1,32 @@
+# The Continous Integration
+
+Arnold uses CircleCI for continous integration.
+
+# Linters
+
+> In development all linters are launched in the `bin/lint` script.
+
+## Dockerfile Linter
+[Haskell Dockerfile Linter](https://github.com/hadolint/hadolint) helps us to
+build a [best practice](https://docs.Docker.com/develop/develop-images/Dockerfile_best-practices/)
+Docker image.
+
+It also uses [ShellCheck](https://github.com/koalaman/shellcheck)
+to lint the Bash code inside `RUN` instructions.
+
+> In development this lint is executed before building Arnold Docker image.
+
+## Shell Linter
+[ShellCheck](https://github.com/koalaman/shellcheck) is a GPLv3 tool that gives warnings and suggestions for bash/sh shell scripts.
+
+See the [gallery of bad code](https://github.com/koalaman/shellcheck/blob/master/README.md#user-content-gallery-of-bad-code) for examples of what ShellCheck can help you identify!
+
+> This Linter is included in Arnold's Docker image.
+> In development this lint is executed after building Arnold Docker image.
+
+## Ansible Linter
+[ansible-lint](https://github.com/willthames/ansible-lint) checks playbooks for practices and behaviour that could potentially be improved.
+We use also [more rules](https://github.com/tsukinowasha/ansible-lint-rules).
+
+> This Linter is included in Arnold's Docker image.
+> In development this lint is executed after building Arnold Docker image.
```

Comment:
I've refactored a bit this documentation:

    # Continuous Integration

    Arnold uses CircleCI for its continuous integration workflow. We will describe
    here quality checks that are performed on this repository.

    ## Linters

    You can run all linters thanks to the `bin/lint` script. By default all linters
    will run on Arnold's repository, but you can choose to run one specific linter
    at a time by giving the linter name as an argument (more details in sections
    below):

    ```bash
    # Run all checks
    $ bin/lint

    # Check Dockerfile
    $ bin/lint docker

    # Check Bash scripts
    $ bin/lint bash

    # Check Ansible playbooks
    $ bin/lint ansible
    ```

    ### Dockerfile

    [Haskell Dockerfile Linter](https://github.com/hadolint/hadolint) helps us to
    build a Docker image following official [best
    practices](https://docs.Docker.com/develop/develop-images/Dockerfile_best-practices/).

    It also uses [ShellCheck](https://github.com/koalaman/shellcheck) (see below) to
    lint the Bash code inside the `RUN` instructions.

    ### Shell

    [ShellCheck](https://github.com/koalaman/shellcheck) is a GPLv3 tool that gives
    warnings and suggestions for bash/sh shell scripts.

    ### Ansible

    [ansible-lint](https://github.com/willthames/ansible-lint) checks playbooks for
    practices and behavior that could potentially be improved. We use also [more
    rules](https://github.com/tsukinowasha/ansible-lint-rules).

WDYT?

Reply:
Ok